service/src/test/java/aero/minova/cas/service/ViewServiceBaseTest.java

```diff
@@ -57,19 +60,57 @@ public abstract class ViewServiceBaseTest<T extends ViewServiceInterface> extend
 	@Autowired
 	AuthoritiesRepository authoritiesRepository;
 
+	@Autowired
+	LuUserPrivilegeUserGroupService luUserPrivilegeUserGroupService;
+
+	@Autowired
+	UserGroupService userGroupService;
+
+	@Autowired
+	ColumnSecurityRepository columnSecurityRepository;
+
 	@PostConstruct
 	void setupViewServiceTest() {
 
 		// Erlaubnis auf Tabelle geben
 		authorizationService.findOrCreateUserPrivilege("xtcasAuthorities");
 		authorizationService.createOrUpdateAdminUser("admin", "$2a$10$l6uLtEVvQAOI7hOXutd7Ye0FtlaL7/npwGu/8YN31EhkHT0wjdtIq");
 
+		// Erlaubnis auf ColumnSecurity Tabelle mit RowLevelSecurity aktiviert
+		LuUserPrivilegeUserGroup privilege = authorizationService.findOrCreateLuUserPrivilegeUserGroup(userGroupService.findEntitiesByKeyText("admin").get(0),
+				authorizationService.findOrCreateUserPrivilege("xtcasColumnSecurity"));
+		privilege.setRowLevelSecurity(true);
+		luUserPrivilegeUserGroupService.save(privilege);
+
 		// Ein paar Daten erstellen
 		authoritiesRepository.save(new Authorities(1, "User1", "test", "user", Timestamp.valueOf("2023-06-18 00:00:00.0").toLocalDateTime(), 1));
 		authoritiesRepository.save(new Authorities(2, "User2", "TEST", "user", Timestamp.valueOf("2023-06-19 00:00:00.0").toLocalDateTime(), 2));
 		authoritiesRepository.save(new Authorities(3, "User3", "not test", "user", Timestamp.valueOf("2023-06-19 08:00:00.0").toLocalDateTime(), 1));
 		authoritiesRepository.save(new Authorities(4, "User4", "testtest", "user", Timestamp.valueOf("2023-06-19 16:00:00.0").toLocalDateTime(), 1));
 		authoritiesRepository.save(new Authorities(5, "User5", "te", null, Timestamp.valueOf("2023-06-20 08:00:00.0").toLocalDateTime(), -1));
+
+		columnSecurityRepository.save(new ColumnSecurity(1, "cs1", "cs1", "cs1", null));
+		columnSecurityRepository.save(new ColumnSecurity(2, "cs2", "cs2", "cs2", "admin"));
+		columnSecurityRepository.save(new ColumnSecurity(3, "cs3", "cs3", "cs3", "niemand"));
```

Comment:
Hier wurde zwar die Column-Security hinzugefügt, aber sie wird nirgendwo abgefragt.

Reply:
In dem Test wird nicht die ColumnSecurity getestet. Die Tabelle wird nur benutzt, weil sie eine SecurityToken Spalte hat, um die RowLevelSecurity zu überprüfen.